src/pages/Account/Components/AnsBanner.tsx

```diff
@@ -0,0 +1,106 @@
+import {
+  Collapse,
+  Alert,
+  Stack,
+  Typography,
+  Divider,
+  IconButton,
+  Button,
+  Box,
+  SxProps,
+  Theme,
+} from "@mui/material";
+import React, {useState} from "react";
+import {AnsConnector} from "@aptos-labs/aptos-name-connector";
+import "@aptos-labs/aptos-name-connector/dist/index.css";
+import CloseIcon from "@mui/icons-material/Close";
+import {grey} from "../../../themes/colors/aptosColorPalette";
+import AptosBannerImage from "../../../assets/Banner.jpg";
+import {useGetInDevMode} from "../../../api/hooks/useGetInDevMode";
+import useSubmitTransaction from "../../../api/hooks/useSubmitTransaction";
+import {useWallet} from "@aptos-labs/wallet-adapter-react";
+export function AnsBanner() {
+  const [bannerOpen, setBannerOpen] = useState<boolean>(true);
+  const inDev = useGetInDevMode();
+  const {connected} = useWallet();
+  const {submitTransaction} = useSubmitTransaction();
+
+  return inDev ? (
```

Comment:
feel free to change the base `Banner` component to accommodate this ANSBanner here: https://github.com/aptos-labs/explorer/blob/46333dc7d62e8d4849cf98fcfdfa1d55b8e915ab/src/components/Banner.tsx#L25 so that we can reuse the base component!